h2o-extensions/xgboost/src/main/java/hex/tree/xgboost/rabit/RabitTrackerH2O.java

```diff
@@ -71,9 +71,8 @@ public boolean start(long timeout) {
             throw new IllegalStateException("Rabit tracker already started.");
         }
         RabitTrackerH2OThread trackerThread = new RabitTrackerH2OThread(this);
-        trackerThread.setDaemon(true);
```

Comment:
There is no reason for this thread to be a daemon thread. It should always be killed in time. And if JVM with H2O is being stopped, the thread should be stopped too.